repositories/productRepository.go

```diff
@@ -177,6 +189,18 @@ func (repository *productRepository) AddProduct(product *models.Product) error {
 		return err
 	}
 
+	message := make(map[string]interface{}, 2)
+
+	message["operation"] = kafka_helpers.OperationEnumString(kafka_helpers.Created)
+	message["product"] = product
+
+	jsonMessage, err := json.Marshal(message)
+	if err != nil {
+		log.Println("An error has occured during marshaling the product, ", err)
+	}
+
+	repository.kafka.WriteMessage(string(jsonMessage), product.ID)
```

Comment:
I thing error check is needed here. If publish is not successful, commit to postgresql shouldn't be called.